### ISSUE TITLE
sphinx html output has all figures filled in with the same single png

### ISSUE THREAD

**Issue by parmentelat** [chrisjsewell/ipypublish#88]
# Bug Report

## Describe the bug

I have a rather long notebook based on the calypso-bash kernel (not that it should matter)

when I run this:

```
nbpublish -f sphinx_ipypublish_main.run -o /Users/tparment/git/flotpython-gittutorial/notebooks/exporting exporting/ALL.ipynb
```

the output html exhibit a very odd behaviour, which is that all the markdown cells that include an image with s/t like `![](media/imagename.png)`, well all these figures show the same image, which is indeed one of the ones that is inserted, but it should not be the same one over and over again.

## To Reproduce

1. step 1
2. step 2
 
(for extended explanation, use the additional context section)

## Minimal Notebook Example

If possible, please create a minimal Jupyter Notebook to replicate the issue.
Then rename its file extension to `.txt` (since Github rejects .ipynb),
and drag and drop it here.

## Runtime Information

(please complete the following information)

- IPyPublish: [e.g. 0.8.3]
- Python: [e.g. 3.6.1]
- OS: [e.g. OSX 10.5.1]
- Pandoc: [run `pandoc --version`]

- (optional for pdf issues) texlive: [run `tex`]
- (optional for pdf issues) latexmk: [run `latexmk --version`]

- Versions of any other relevant python packages

## Logging

Drag and drop the log file here, or copy and paste the output from the terminal

**Comment by chrisjsewell** [comment]
@parmentelat good spot :) This should be fixed via #89, but feel free to reopen if you still have any issues. I'll release a new version shortly

**Comment by parmentelat** [comment]
thanks for fixing it :)  
do you have any estimate for that release ? I'd rather wait for it than install from sources, if that's likely to happen shortly; what I do in these cases is to issue a dev release so that others can test by installing with `pip install --pre`, maybe this scheme could work here too 

thanks !

**Comment by chrisjsewell** [comment]
Already released `v0.10.4` yesterday :) FYI, if you didn't know, ipypublish is also available on conda-forge: https://anaconda.org/conda-forge/ipypublish, which is the recommended way of installing
